<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CRtmpPublish</class>
 <widget class="QDialog" name="CRtmpPublish">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>674</width>
    <height>652</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>20</y>
     <width>511</width>
     <height>624</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <property name="fieldGrowthPolicy">
     <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Primary Server</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QCheckBox" name="m_PrimaryEnable">
      <property name="text">
       <string>Enable</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Address</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="m_PrimaryServer"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_14">
      <property name="text">
       <string>Port</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="m_PrimaryPort"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Application</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="m_PrimaryApplication"/>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Stream</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="m_PrimaryStream"/>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>User Id</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="m_PrimaryUserId"/>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QLineEdit" name="m_PrimaryPassword"/>
    </item>
    <item row="17" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Secondary Server</string>
      </property>
     </widget>
    </item>
    <item row="17" column="1">
     <widget class="QCheckBox" name="m_SecondaryEnable">
      <property name="text">
       <string>Enable</string>
      </property>
     </widget>
    </item>
    <item row="18" column="0">
     <widget class="QLabel" name="label_12">
      <property name="text">
       <string>Address</string>
      </property>
     </widget>
    </item>
    <item row="18" column="1">
     <widget class="QLineEdit" name="m_SecondaryServer"/>
    </item>
    <item row="19" column="0">
     <widget class="QLabel" name="label_13">
      <property name="text">
       <string>Port</string>
      </property>
     </widget>
    </item>
    <item row="19" column="1">
     <widget class="QLineEdit" name="m_SecondaryPort"/>
    </item>
    <item row="20" column="0">
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>Application</string>
      </property>
     </widget>
    </item>
    <item row="20" column="1">
     <widget class="QLineEdit" name="m_SecondaryApplication"/>
    </item>
    <item row="21" column="0">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Stream</string>
      </property>
     </widget>
    </item>
    <item row="21" column="1">
     <widget class="QLineEdit" name="m_SecondaryStream"/>
    </item>
    <item row="22" column="0">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>User Id</string>
      </property>
     </widget>
    </item>
    <item row="22" column="1">
     <widget class="QLineEdit" name="m_SecondaryUserId"/>
    </item>
    <item row="23" column="0">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
    </item>
    <item row="23" column="1">
     <widget class="QLineEdit" name="m_SecondaryPassword"/>
    </item>
    <item row="15" column="0" colspan="2">
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="31" column="1">
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CRtmpPublish</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CRtmpPublish</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
